src/main/java/com/onarandombox/MultiverseCore/destination/BedDestination.java

```diff
@@ -23,7 +23,7 @@
  * A bed-{@link MVDestination}.
  */
 public class BedDestination implements MVDestination {
-    public static final String OLD_BED_STRING = "b:playerbed";
+    public static final String OLD_BED_STRING = "playerbed";
```

Comment:
`OLD_BED_STRING` doesnt sound like a good variable name to me. Something like `OWN_BED_STRING` would be preferred.